helm_charts/stunner_helm/values.yaml

```diff
@@ -0,0 +1,105 @@
+# Default values for stunner_helm.
+
+turnServer:
+  namespace: default
+  deployment:
+    name: stunner
+    label: stunner
```

Comment:
Can we set a default replica size? I'd say let's just set this to 1 by default, but let's just allow the user to customize this.